tests/windows_test.go

```diff
@@ -161,8 +164,11 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 			})
 		})
 
-		Context("VMI with subdomain is created", func() {
-			BeforeEach(func() {
+		Context("[Serial] VMI with subdomain is created", Ordered, func() {
```

Comment:
We run all the tests that have `[Serial]` in one process after the parallel runs are finished
so theoretically it does affect, because it has global effect.

But since `Ordered` are running in their own process even when running serial
The right thing to do IIUC is:
1. Don't add Serial here
and instead:
2. If possible combine all the contexts that are possible into one Ordered context
else it each context will be on its own still isn't it ?
(assuming it is possible to combine contexts here, please check about it if possible)

https://onsi.github.io/ginkgo/MIGRATING_TO_V2#ordered-decorator

Reply:
Since these two `Contexts` (this and on line 196) create Windows with different bindings, so we need to be re-creating the VMs. So it seems we can't merge specs from these `Context`s in to a single `Ordered Context`, and as there's only one `It`, I see no gain in making it `Ordered`.

Reply:
Thanks for explaining how `[Serial]` works. I will remove it.

Reply:
I would keep the `Ordered` decorator. It does not change behavior, but will be useful if a future PR adds more tests.

Reply:
It makes it unable to run in parallel with other `Ordered` contexts in the same parallel process